# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cookie-store/cookieStore_set_arguments.https.any.js.ini
# Commit: bb99a3d0d1e7
# Full Hash: bb99a3d0d1e7bf2278834e61e8878440bca791e3
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2025-03-29 09:19:43
# Regressor Bug: 1951750
# File Overlap Count: 4
# Description:
#   Bug 1951750 - Handle domain argument in CookieStoreParent r=baku,edgul,cookie-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D240695
# ==============================================================================

diff -r ee233c9ac108 -r bb99a3d0d1e7 testing/web-platform/meta/cookie-store/cookieStore_set_arguments.https.any.js.ini
--- a/testing/web-platform/meta/cookie-store/cookieStore_set_arguments.https.any.js.ini	Fri Mar 28 13:48:31 2025 +0000
+++ b/testing/web-platform/meta/cookie-store/cookieStore_set_arguments.https.any.js.ini	Fri Mar 28 13:48:31 2025 +0000
@@ -1,14 +1,8 @@
 [cookieStore_set_arguments.https.any.html]
-  [cookieStore.set default domain is null and differs from current hostname]
-    expected: FAIL
-
   [cookieStore.set adds / to path that does not end with /]
     expected: FAIL
 
 
 [cookieStore_set_arguments.https.any.serviceworker.html]
-  [cookieStore.set default domain is null and differs from current hostname]
-    expected: FAIL
-
   [cookieStore.set adds / to path that does not end with /]
     expected: FAIL